src/pages/experimental/create-flows/index.mdx

```diff
@@ -141,6 +141,16 @@ If your creation flow consists of multiple fields and/or supporting imagery, con
 
 ## Anatomy
 
+<AnchorLinks>
```

Comment:
```suggestion
<AnchorLinks small>
```

This gives you smaller anchor links for in-line use

Reply:
awesome, I should have guessed there was something like this available!